neonvm/main.go

```diff
@@ -157,27 +159,32 @@ func main() {
 		MaxConcurrentReconciles: concurrencyLimit,
 	}
 
-	if err = (&controllers.VirtualMachineReconciler{
+	virtualmachineReconciler := &controllers.VirtualMachineReconciler{
```

Comment:
nit: We have to start writing `vm` instead of `virtualmachine` in code at some point. I know the full name is all over the codebase and it will look inconsistent. But it is as if this file has not many of those, so maybe a good place to start.

Reply:
Fair enough - handled that as part of 23148bdd35e7b7e3fba12d756bd47176c2c8cd71